.studio/common

```diff
@@ -251,8 +251,10 @@ function verify_component() {
 }
 
 function display_available_components() {
-  log_line "Available components:"
-  ls -1 /src/components | awk '{print "* "$1}'
+  log_line "Available targets:"
+  echo "  * api"
+  echo "  * config"
+  ls -1 /src/components/*/scripts/grpc.sh | awk -F/ '{ print "  * "$4 }'
```

Comment:
display_available_components is called from verify_component which is called from a number of places that accept many more components than the ones this will generate.  Perhaps we need two different functions?

Reply:
When you're right, you're right!

Good catch, let me rewrite it with another function.

Reply:
I pushed a commit with a new method for this: `display_valid_protobuf_targets`